Pod/Classes/Form/PDFFormViewController.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-open class PDFFormViewController: NSObject {
+open class PDFFormViewController {
```

Comment:
@pushchris conformance to NSObject has been restored with https://github.com/mytrus/UXMPDFKit/commit/e84ac2670901a4aac2a3d0b62e188eaee401d287